backend/main.py

```
@@ -218,31 +222,32 @@ class ReadinessResponse(BaseModel):
         classifier_service.load()
     except FileNotFoundError as e:
-        print(f"[WARNING] Classifier not loaded: {e}")
+        logger.warning("Classifier not loaded: %s", e)
     try:
         ner_service.load()
     except FileNotFoundError as e:
-        print(f"[WARNING] NER not loaded: {e}")
+        logger.warning("NER not loaded: %s", e)
     try:
         duplicate_service.load()
     except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)